change/@fluentui-react-components-de1ae615-fbbf-4fd4-8957-40d69ff9414f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Patch will trigger `9.0.0` release, it is okay of do you want to stay in alpha for now?